.circleci/config.yml

```diff
@@ -31,18 +31,22 @@ jobs:
       - run:
           name: Compile test contracts
           command: ./scripts/compile_contracts.sh
+      - run:
+          name: Check out previous test metadata
+          command: | 
+            cat "${CIRCLE_INTERNAL_TASK_DATA}/circle-test-results/results.json" | jq .
       - run:
           name: Run pytest tests
           parallel: true
           command: |
-            TESTFILES=$(
-                circleci tests glob test/test*.py |
-                circleci tests split --split-by=timings --timings-type=classname
-            )
             mkdir -p test-results
-            poetry run pytest -s --junitxml=test-results/junit.xml -vv $TESTFILES
+            SPLITED_FILES_CLASS=$(circleci tests glob test/test*.py | sed  "s/\.py//g; s/\//./g" | circleci tests split --split-by=timings --timings-type=classname)
+            TEST_FILES=$(sed "s/\./\//g; s/ /.py /g; s/$/.py/g;" \<<< $SPLITED_FILES_CLASS)
+            poetry run pytest --verbose --junitxml=test-results/junit.xml $TEST_FILES
```

Comment:
You've modified the `poetry run pytest` command:
- Can you restore the `-s` flag?
- Is there any difference between `-vv` and `--verbose`?

Reply:
I will restore -s -vv, as -vv is double verbose aka. max verbose